.github/workflows/release.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - uses: risc0/cargo-install@9f6037ed331dcf7da101461a20656273fa72abf0
         with:
           crate: cargo-binstall
-          version: "1.4"
+          version: "1.10.15"
```

Comment:
to do in the future: drop binstall and use rzup to download cargo-risczero

Reply:
1.4 doesn't compile using rust 1.81 so I'm bumping it for now. this version build successfully when I tested locally.